glide.lock

```diff
@@ -23,4 +23,16 @@ imports:
   - config/validate/report
 - name: github.com/vincent-petithory/dataurl
   version: 9a301d65acbb728fcc3ace14f45f511a4cfeea9c
-testImports: []
+testImports:
```

Comment:
It doesn't look like we're actually pulling in the tests themselves (`glide-vc` is probably removing them). Is there a way to not need this?

Reply:
Added to the `ignore` section of `glide.yaml`, they shouldn't be included anymore.